new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/AppBuilder.java

```diff
@@ -97,6 +97,39 @@ public boolean build() {
     return jarBuilt && zipBuilt;
   }
 
+  /**
+   * Build an application archive using a pre-populated AppParams instance.
+   * @return true if the command succeeds
+   */
+  public boolean buildCoherence() {
```

Comment:
I think you can use buildApplication() in BuildApplication.java for this.

Reply:
same comment above

Reply:
please revisit

Reply:
This method is used to build archive for coherence-proxy-server. After discussing with Bhavani, we both agree to keep it as it is